src/fixate/config/__init__.py

```diff
@@ -20,13 +18,28 @@
 from typing import Dict, List, Optional
 import enum
 import re
+import platformdirs
+from pathlib import Path
+import logging
+
+logger = logging.getLogger(__name__)
+
+CONFIG_DIRECTORY = Path(platformdirs.site_config_dir("Fixate", False))
+LOG_DIRECTORY = Path(platformdirs.user_log_dir("Fixate", False))
+INSTRUMENT_CONFIG_FILE = CONFIG_DIRECTORY / "instruments.json"
 
-LOCAL_CONFIG_PATH = os.path.join(os.path.dirname(__file__), "local_config.json")
+if not CONFIG_DIRECTORY.exists():
+    # Create directory if it doesn't already exist
+    CONFIG_DIRECTORY.mkdir(parents=True, exist_ok=True)
+
+if not LOG_DIRECTORY.exists():
+    LOG_DIRECTORY.mkdir(parents=True, exist_ok=True)
 
```

Comment:
I'd like to avoid any logic running at import time. The log directory check should move into the `if not args.disable_logs:` block in __main__

The CONFIG_DIRECTORY check probably can move to fxconfig? that the only place we attempt to write out a file to that location, I think...

Reply:
I note that there are probably a bunch of counter examples still littered through this code base :)

Reply:
Done now